src/meetings/2021-11-22.md

```diff
@@ -0,0 +1,31 @@
+# 2021-11-22
+
+| Info                                           |                          |
+| ---------------------------------------------- | ------------------------ |
+| Time                                           | [See in your time zone]  |
+| Calendar event (with Zoom details)             | [link][cal]              |
+| Social hour calendar event (with Zoom details) | [link][calsh]            |
+| Recording                                      | (link posted after call) |
+
+[see in your time zone]: https://everytimezone.com/s/91c9791f
+[cal]: https://calendar.google.com/event?action=TEMPLATE&tmeid=NnVibTVzcmEwdnZxMTBtYjZ0ZmZxaWtyYXQgN24wdnZvcWZlMGtibms2aTA0dWl1NTJ0MzBAZw&tmsrc=7n0vvoqfe0kbnk6i04uiu52t30%40group.calendar.google.com
+[calsh]: https://calendar.google.com/event?action=TEMPLATE&tmeid=NGd2N2FzYjBnMHFiNXA4dWdtYWZ2ZmxobmwgN24wdnZvcWZlMGtibms2aTA0dWl1NTJ0MzBAZw&tmsrc=7n0vvoqfe0kbnk6i04uiu52t30%40group.calendar.google.com
+
+## Agenda
+
+The theme for this meeting is Rust in the Linux kernel.
+
+- (5 min) Opening remarks 👋 ([nikomatsakis])
+- (40 min) Rust in Linux ([ojeda])
+- (5 min) Closing ([nikomatsakis])
+
+[nikomatsakis]: https://github.com/nikomatsakis
+[ojeda]: https://github.com/ojeda
```

Comment:
@ojeda, I've added the barebones of your section here. Can I get you to write a bit more of a description, and add the Github accounts of Alex/Wedson if they'll be part of the presentation? Feel free to just write in a comment, and I can add it in if that's easier :smiley:

Reply:
Of course!

> The Rust for Linux project is adding Rust support to the Linux kernel. While it is still the early days, there are some areas of the Rust language, library and tooling where the Rust project might be able to help out -- for instance, via stabilization of features, suggesting ways to tackle particular problems... This talk will walk through the issues found, along with examples where applicable.

The title should probably be "Rust for Linux" (i.e. the name of the project).

The GitHub are @alex and @wedsonaf.

Reply:
@ojeda looks good :) got it into the description. I added @alex and @wedsonaf as well, but we can sort who the primary speaker will be later :+1: